tests/cpp/cache_eviction.cpp

```diff
@@ -202,6 +210,50 @@ const std::vector<LowScoreBlocksTestStruct> LOW_SCORE_BLOCK_EVICTION_TEST_CASES
                 {{0, 9, 10, 11, 13}, {12, 11, 8, 9, 17}},
                 {{8, 9, 10, 11, 13}, {8, 9, 10, 11, 12}}
         },
+
+        // kvcrush enabled cases
+        //low-scored blocks in evictable area
```

Comment:
Please add a tag to each KVCrush-enabled test case comment instead of area effect comments, e.g. `// low-scored blocks in evictable area - with KVCrush`.
Also please remove the test cases for which it is obvious that they will never pass the `bool should_apply_kvcrush` condition in the test case body.